src/studio-home/data/api.test.js

```diff
@@ -53,6 +54,16 @@ describe('studio-home api calls', () => {
     expect(result).toEqual(expected);
   });
 
+  fit('should get studio courses data v2', async () => {
```

Comment:
Why `fit`? My understanding is that this will prevent other tests from running and skip them. At least locally all tests succeed if you replace the `fit`s with `it`. How about trying that? It would mean replacing the fit here and also the one on line 47.

Reply:
I agree, let's change this to `it`